lib/request-worker.js

```diff
@@ -24,17 +24,18 @@ function doRequest() {
 	debug(`child request, cmd size: ${reqData.length} bytes`);
 	const req = JSON.parse(reqData);
 
-	if (!isSecurityEnhancedDomain(req.url)) {
-		process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
-	}
+	const port = req.options !== undefined ? req.options.port : undefined;
+	const rejectUnauthorized = req.options !== undefined ? req.options.rejectUnauthorized : false;
```

Comment:
```suggestion
	const rejectUnauthorized = req.options?.rejectUnauthorized;
```
or even 
```suggestion
const isSecurityEnhanced = req.options?.rejectUnauthorized || isSecurityEnhancedDomain(req.url);
```
what is the oldest node.js version that we are targeting? Even node.js v14 should have no issue with this syntax.